apps/native-component-list/src/api/registerForPushNotificationsAsync.ts

```diff
@@ -48,23 +49,28 @@ export default async function registerForPushNotificationsAsync() {
     ]),
   });
 
-  let result = await response.json();
+  const result = await response.json();
   if (result.errors) {
-    for (let error of result.errors) {
+    for (const error of result.errors) {
+      // tslint:disable-next-line no-console
       console.warn(`API error sending push notification:`, error);
     }
   }
 
-  let receipts = result.data;
+  const receipts = result.data;
   if (receipts) {
-    let receipt = receipts[0];
+    const receipt = receipts[0];
     if (receipt.status === 'error') {
       if (receipt.details) {
+        // tslint:disable-next-line no-console
```

Comment:
Maybe we should disable no-console warning for the whole project?

Reply:
IMO, `console.{logs, warns, errors}` are useful during development and might easily slip through reviews.
That annotation suggest these are used on purpose.

Reply:
I was thinking NCL is to some extend used for development and some `console.sths` are ok in this specific project. (Adding `// tslint` to every `console.log` seems a bit awkward)